src/pages/journal/components/test-outcome/test-outcome.ts

```diff
@@ -10,6 +10,17 @@ import { end2endPracticeSlotId } from '../../../../shared/mocks/test-slot-ids.mo
 import { COMMUNICATION_PAGE, OFFICE_PAGE, PASS_FINALISATION_PAGE } from '../../../page-names.constants';
 import { ModalEvent } from '../../journal-rekey-modal/journal-rekey-modal.constants';
 import { DateTime, Duration } from '../../../../shared/helpers/date-time';
+import { SlotDetail } from '../../../../shared/models/DJournal';
+import { Observable } from 'rxjs/Observable';
+import { Subscription } from 'rxjs/Subscription';
+import { ActivityCode } from '@dvsa/mes-test-schema/categories/B';
+import { getTests } from '../../../../modules/tests/tests.reducer';
+import { getActivityCodeBySlotId } from '../../../../modules/tests/tests.selector';
+import { map } from 'rxjs/operators';
+
+interface TestOutcomeComponentState {
```

Comment:
I believe sub components should be as dumb as possible and should be connected to the store as a last resort. 

Can the slot component itself get the data and pass it in. I don't think any of the sub components of the test slot are connected to the store.

Reply:
Okay only did it this way as it was easier to implement but will change